modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -150,8 +160,25 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
     function _process($values)
     {
         if ($this->_saveFile($values)) {
+            // Save file succeeded. All that's left to do is launch the MRI
+            // pipeline if the auto-launch switch is on
+            $config = NDB_Config::singleton();
+            $ImagingUploaderAutoLaunch = $config->getSetting(
+                'ImagingUploaderAutoLaunch'
+            );
+
+            if ($ImagingUploaderAutoLaunch) {
+                // Perform the real upload on the server
+                $serverProcessLauncher = new ServerProcessLauncher();
+                $serverProcessLauncher->mriUpload(
```

Comment:
Makes sense to me to perform the auto-launch in the process function (as opposed to importFile function)